tests/installation/common_criteria_configuration/cc_disk_encryption_password.pm

```diff
@@ -0,0 +1,19 @@
+# Copyright SUSE LLC
```

Comment:
cc is not well known by others, it would better to use "common_criteria" and a shorten a bit the rest.